src/content/docs/recipes/toggle-between-dark-and-light-theme.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Toggle Dark & Light Theme
+---
+
+With help of this, you will understand basic concept of implementing dark and light theme in your Astro project.
+
+Now, we can start writing javascript code, for:
+    - Getting the current theme from local storage or system preference
+    - Toggle the value of theme, and store it in local storage
+    - Reflecting the theme in the document
+    - Listening for clicks on the button to toggle the theme
+    - Listening for system changes to update the theme 
+    - Listening for ViewTransition event to update the theme on page change ❌
+
+Getting the current theme from local storage or system preference
+
+```html
+<script>
```

Comment:
Not a big fan of having all these parts using `<script>` and then right at the end in the full code having `<script is:inline>`.

I feel either all these code snippets need to also use `is:inline`, or just have a paragraph before any code that just says basically "All the code for this needs to be `is:inline` to avoid FOUC, and placed after your controls in the DOM so they can be found by js blah blah" yknow just something!

I think someone trying to read this quickly will miss that!